Documentation/ApiOverview/Database/DatabaseStructure/Index.rst

```diff
@@ -1,124 +1,135 @@
-table records may only be created on a real page.
-
-.. index::
-   Tables; MM relations
-   Tables; Cache
-   Tables; System information
-.. _database-structure-other-tables:
+The :sql:`pages` table has a special status: It is the backbone of TYPO3, as it
+provides the hierarchical page structure into which all other records managed by
+TYPO3 are placed. All other managed tables in TYPO3 have a :sql:`pid`
+field that points to a :sql:`uid` record in this table. Thus, each managed table
+record in TYPO3 is always placed on exactly one page in the page tree. This
+makes the :sql:`pages` table the mother of all other managed tables. It can be
+seen as a directory tree with all other table records as files.
+
+Standard pages are literally website pages in the frontend. But they can also be
+storage spaces in the backend, similar to folders on a hard disk. For each
+record, the :sql:`pid` field contains a reference to the page where that
+record is stored. For pages, the :sql:`pid` fields behaves as a reference to
+their parent pages.
+
+The special "root" page has some unique properties: its :sql:`pid` is 0 (zero),
+it does not exist as a row in the :sql:`pages` table, only users with
+administrative rights can access records on it, and these records must be
+:ref:`explicitly configured to reside in the root page
+<t3tca:ctrl-reference-rootlevel>` - usually, table records can only be created
+on a real page.
+
+..  index::
+    Tables; MM relations
+    Tables; Cache
+    Tables; System information
+..  _database-structure-other-tables:
 
 Other Tables
 ============
 
-The tables which are not managed via the TYPO3 CMS backend fill various
-roles. Some of the most common are:
-
-- MM relations: when tables are related using a many-to-many relationship,
-  another table must hold these relations. Examples are the table storing
-  relations between categories and categorized records
-  ("sys\_category\_record\_mm") or the table storing relations
-  between files and their various usages in pages, content elements, etc.
-  ("sys\_file\_reference"). The latter is an interesting example,
-  because it does actually appear in the backend, although only
-  as part of :ref:`inline records <t3tca:columns-inline>`.
-
-- Cache: when a cache is defined as using the database as a cache backend,
-  TYPO3 CMS will automatically create and manage the relevant cache tables.
-
-- System information: there exist tables storing information about sessions,
-  both frontend and backend ("fe\_sessions" and "be\_sessions" respectively),
-  a table for a central registry ("sys\_registry") and quite a few others.
-
-All these tables are not subject to the uid/pid constraint mentioned
-above, but they may have such fields if it is convenient for whatever
+The tables which are not managed through the TYPO3 backend serve various
+purposes. Some of the most common are:
+
+-   **MM relations:** When tables are connected via a many-to-many relationship,
```

Comment:
Use a definition list for added semantics to the markup?

Reply:
Nope, there are just three list points which are easy to grasp. Don't think it is necessary.

Reply:
You are making something bold here that has the function of a headline or definition. We should avoid bold unless for emphasis

Reply:
Okay, was for better readibility. Will remove it.